programs/ssl/ssl_test_common_source.c

```diff
@@ -263,12 +263,24 @@ int send_cb( void *ctx, unsigned char const *buf, size_t len )
 
 #if defined(MBEDTLS_X509_CRT_PARSE_C)
 #if defined(MBEDTLS_ECDSA_C) && defined(MBEDTLS_RSA_C)
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3)
```

Comment:
We should not have to change anything here. This is related to the allow_sha1 option of ssl_client/server2 that should be equal to 0 for TLS 1.3 only tests.

Reply:
Add comments about that. And allow_sha1 option is enabled for TLS1.3 now.

Reply:
I add some tests with/without `allow_sha1`.  
Here is not for TLS1.3 only build.

Reply:
@ronald-cron-arm should we disable allow_sha1 in `ssl_conf_check`?

Reply:
Agree. I also think about it. It should be moved out of this PR